src/main/java/seedu/guest/model/guest/Guest.java

```diff
@@ -20,20 +20,22 @@ public class Guest {
     private final NumberOfGuests numberOfGuests;
     private final IsRoomClean isRoomClean;
     private final Bill bill;
+    private final Request request;
 
     /**
      * Every field must be present and not null.
      */
     public Guest(Name name, Phone phone, Email email, DateRange dateRange,
-                 NumberOfGuests numberOfGuests, IsRoomClean isRoomClean, Bill bill) {
-        requireAllNonNull(name, phone, email, dateRange, numberOfGuests, isRoomClean, bill);
+                 NumberOfGuests numberOfGuests, IsRoomClean isRoomClean, Bill bill, Request requests) {
```

Comment:
Should this requests be request since we use request in other parts of the program?

Reply:
changed